.env

```
@@ -9,4 +9,12 @@ NEXT_PUBLIC_VAULTS_APP_URL=https://vaults.secured.finance
 NEXT_PUBLIC_SHOW_STABLECOIN_APP_URL=true
 NEXT_PUBLIC_REFERRAL_MESSAGE="Join the @Secured_Fi Points Program! ✨\n\nSign up through the link below and receive 50 points as a welcome gift! Plus, enjoy a 5% boost on points earned. 🎁\n\n%URL%?ref=%REF%\n\nComplete various quests and earn chances for future \$SFC airdrops! 🪂\n"
 NEXT_PUBLIC_SQUID_WIDGET_INTEGRATOR_ID=
-NEXT_PUBLIC_GOOGLE_ANALYTICS_TAG=
\ No newline at end of file
+NEXT_PUBLIC_GOOGLE_ANALYTICS_TAG=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_GOOGLE_ANALYTICS_TAG key should go before the NEXT_PUBLIC_GRAPHQL_SERVER_URL key

(UnorderedKey)